QingStorSDK.xcodeproj/project.pbxproj

```diff
@@ -355,7 +352,7 @@
 			isa = PBXProject;
 			attributes = {
 				LastSwiftUpdateCheck = 0810;
```

Comment:
This code is automatically generated, I think it should not be manually changed.